app_dart/lib/src/request_handlers/dart_internal_subscription.dart

```diff
@@ -66,7 +66,8 @@ class DartInternalSubscription extends SubscriptionHandler {
 
     // Only publish the parent release_builder builds to the datastore.
     // TODO(drewroengoogle): Determine which builds we want to save to the datastore and check for them.
-    final regex = RegExp(r'(Linux|Mac|Windows)\s+(engine_release_builder|packaging_release_builder)');
+    final regex =
+        RegExp(r'(Linux|Mac|Windows)\s+(engine_release_builder|packaging_release_builder|flutter_release_builder)');
```

Comment:
Keyong had a good suggestion for me a while back while trying to do this. It might be better to add a property to the builder and use that. That way if someone changes the name a bit or something like that this won't break.

Reply:
That's a good point. However, I am going to update the TODO on line 68 because there is an additional request to eventually handle *all* dart-internal builds instead of just the ones in this regex, making this regex not necessary in the long term once that has landed.

Reply:
I would suggest then linking the issue that tracks that work here. Is that on the work queue or is that something that is planned for the future?

Reply:
https://github.com/flutter/flutter/issues/134674 is the issue I just created for it under the project https://github.com/orgs/flutter/projects/74, as it was a recently requested feature. It is something in my work queue that I am intending on completing soon as it will make it easier for release engineers to diagnose failures.

Reply:
Okay cool! Thanks!